laws/js/src/main/scala/cats/laws/Platform.scala

```diff
@@ -0,0 +1,15 @@
+package cats
+package laws
+
+import org.scalacheck.{Arbitrary, Prop}
+import org.scalacheck.Prop._
+import Prop.{False, Proof, Result}
+
+private[laws] object Platform {
+
+  // Scala-js does not implement the Serializable interface, so we just retuen true.
```

Comment:
Can fix now, but if this is the only issue, I would rather add to https://github.com/non/cats/issues/429.

Well spotted, though!